mappings/net/minecraft/world/entity/EntityStore.mapping

```diff
@@ -0,0 +1,17 @@
+CLASS net/minecraft/class_5577 net/minecraft/world/entity/EntityStore
```

Comment:
EntityStorage, perhaps? Calling it just "EntityStore" seems wrong to me

Reply:
Too Grondagian?

Reply:
er, explain that to me please 😅

Reply:
Grondag's [Fluidity library](https://github.com/grondag/fluidity) has objects called `Store`s that can store various things.